examples/flutter_gallery/.idea/workspace.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We _do_ want the template's workspace.xml file committed, so we can pre-populate projects with things like opening main.dart. We don't want the workspace.xml committed for active projects - we'll just see a lot of SCM churn and sharing of per-user, per-machine settings.